### PR TITLE
Basic styling

### PR DESCRIPTION
**Pull request by celinevo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,24 +5,151 @@
...
+  </style> 
 </head>
 <body>
 <header>
     <h1>Celine Vogel</h1>
-    <nav>
-        <a href="#contact">Contact</a>
-        <a href="#biography">Biography</a>
+
+    <br>
```
Comment (lalueders) [inline index.html]: Not sure here, but I would not use `<br>` for styling html. Instead I would solve this with CSS.

**Review by lalueders — APPROVED** [review]
Good job! Check my comment ;)

File: index.html [hunk 2]
```diff
@@ -5,24 +5,151 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Celine Vogel Personal Website</title>
+<link rel="stylesheet" href="styles.css">
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300&display=swap');
+  </style>
```
Comment (mwarnck) [inline index.html]: check the indentation ;)

File: index.html [hunk 3]
```diff
@@ -5,24 +5,151 @@
...
+
 <figure>
-<img src="https://source.unsplash.com/random/600x300" alt="">
-<figcaption>Random image, that changes every time you reload.</figcaption>
+    <img src="https://photostockeditor.com/asset/1080/35/359a/PSE371.jpg" class="image" alt="Picture of a Laptop in the dark.">
 </figure>
-<section id="contact">
-  <h2>Contact</h2>
+
+<br>
```
Comment (mwarnck) [inline index.html]: I think Jerry told us not to use this form for spaces and should solve this with CSS :)

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    20% { transform: translate(-3px, 0px) rotate(1deg); }
+    30% { transform: translate(3px, 2px) rotate(0deg); }
+    40% { transform: translate(1px, -1px) rotate(1deg); }
+    50% { transform: translate(-1px, 2px) rotate(-1deg); }
+    60% { transform: translate(-3px, 1px) rotate(0deg); }
+    70% { transform: translate(3px, 1px) rotate(-1deg); }
+    80% { transform: translate(-1px, -1px) rotate(1deg); }
+    90% { transform: translate(1px, 2px) rotate(0deg); }
+    100% { transform: translate(1px, -2px) rotate(-1deg); }
+  }
```
Comment (mwarnck) [inline styles.css:58]: I have no idea what this is doing, haha

**Review by mwarnck — COMMENTED** [review]
overall looks good for me. only the indentation should be checked and the spaces done with the br element ;)

File: index.html (hunk 1 above)
Comment (celinevo) [reply index.html]: Your are right, I am going to change it 🗡️

File: index.html (hunk 3 above)
Comment (celinevo) [reply index.html]: Yes I totally forgot. Thanks for letting me know!

**Pull request merged**